### PR TITLE
add 0913 meetup page

### PR DESCRIPTION
**Pull request by jamie212**
I added the information for the 09/13 meetup in 0913-nycu.rst and added an entry in index.rst.

### CODE REVIEW

**Comment by kuomc** [comment]
I had create the kktix sign up page. The link is https://sciwork.kktix.cc/events/meetup-20230913, same as the link in your page.

**Comment by jamie212** [comment]
> I had create the kktix sign up page. The link is https://sciwork.kktix.cc/events/meetup-20230913, same as the link in your page.

Thanks! However it seems the date in the time section was entered incorrectly?

**Comment by kuomc** [comment]
> > I had create the kktix sign up page. The link is https://sciwork.kktix.cc/events/meetup-20230913, same as the link in your page.
> 
> Thanks! However it seems the date in the time section was entered incorrectly?

Thanks, I've corrected it.

File: content/pages/meetup/2023/0913-nycu.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
+========================================
+Meetup 2023 September 6th in NYCU
```
Comment (yungyuc) [inline content/pages/meetup/2023/0913-nycu.rst:2]: @jamie212 , the date in the subject is incorrect.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: content/pages/meetup/2023/0913-nycu.rst (hunk 1 above)
Comment (jamie212) [reply content/pages/meetup/2023/0913-nycu.rst]: > @jamie212 , the date in the subject is incorrect.

Thank you for pointing it out, I've corrected it.

File: content/pages/meetup/2023/0913-nycu.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+
+Preparational meeting for September scisprint
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+Additionally, we will discuss the following items for preparing September scisprint. 
+
+* Task arrangement for volunteers
+* Promotion plan for September scisprint
+
```
Comment (kuomc) [inline content/pages/meetup/2023/0913-nycu.rst:46]: Please help me add the link of scisprint and kkitx. The sentance will look like : 
Please check September scisprint (https://sciwork.dev/sprint/2023/09-hsinchu) and kktix (https://sciwork.kktix.cc/events/scisprint-202309-hsinchu) for more information.

Comment (kuomc) [reply content/pages/meetup/2023/0913-nycu.rst:46]: Please use hyberlink and only display September scisprint and kktix.

Comment (jamie212) [reply content/pages/meetup/2023/0913-nycu.rst:46]: Okay, I've made the changes.

**Pull request merged**